From 297c734c0b97e26e2c3b4322f9ce9947b08fb3a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20Dr=C4=85g?= <piotrdrag@gmail.com>
Date: Sat, 28 Jan 2017 23:57:01 +0100
Subject: [PATCH 2/2] Add Keywords to the .desktop file

It's a part of the Desktop Entry Specification [1] and it's used by,
among others, the search system in GNOME 3.

The comment is there to hopefully minimize the chance of translators
making a mistake that could potentially break build.

[1] https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html
---
 gtk/transmission-gtk.desktop.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gtk/transmission-gtk.desktop.in b/gtk/transmission-gtk.desktop.in
index 2b6e799b8..3e606264f 100644
--- a/gtk/transmission-gtk.desktop.in
+++ b/gtk/transmission-gtk.desktop.in
@@ -2,6 +2,8 @@
 _Name=Transmission
 _GenericName=BitTorrent Client
 _Comment=Download and share files over BitTorrent
+# Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+_Keywords=torrents;downloading;uploading;share;sharing;
 Exec=transmission-gtk %U
 Icon=transmission
 Terminal=false
-- 
2.14.3

